src/main/resources/view/MainWindow.fxml

```diff
@@ -25,36 +27,47 @@
 
       <VBox>
         <MenuBar fx:id="menuBar" VBox.vgrow="NEVER">
-          <Menu mnemonicParsing="false" text="File">
-            <MenuItem mnemonicParsing="false" onAction="#handleExit" text="Exit" />
+          <Menu fx:id="menuHelpTab" mnemonicParsing="false" text="Help">
+            <MenuItem fx:id="helpMenuItem" styleClass="menu-item" mnemonicParsing="false" onAction="#handleHelp" text="Help" />
+          </Menu>
```

Comment:
I don't think the help menu should be grouped with the other options as the other options are more of tabs while the help menu is a menu.

Reply:
See https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/75#discussion_r501141034.